src/cdk/testing/change-detection.ts

```diff
@@ -110,13 +110,63 @@ export async function manualChangeDetection<T>(fn: () => Promise<T>) {
   return batchChangeDetection(fn, false);
 }
 
+
+
+/**
+ * Resolves the given list of async values in parallel (i.e. via Promise.all) while batching change
+ * detection over the entire operation such that change detection occurs exactly once before
+ * resolving the values and once after.
+ * @param values A getter for the async values to resolve in parallel with batched change detection.
+ * @return The resolved values.
+ */
+export function parallel<T1, T2, T3, T4, T5>(
```

Comment:
Oh good catch, is there some way to do this without having a separate declaration for each number of params?

This is what I get for putting off actually using the function in our own tests 😞

Reply:
There isn't. I took this from TS's built-in type for `Promise.all` and I've seen rxjs do something similar.